modal/_clustered_functions.py

```diff
@@ -0,0 +1,56 @@
+from modal._utils.async_utils import synchronize_api
+
+
+async def _initialize_clustered_function(cluster_id: str, cluster_rank: int, cluster_size: int):
+    print(f"Initializing clustered function with cluster ID {cluster_id}, rank {cluster_rank}, size {cluster_size}")
```

Comment:
Temporary placeholder. I plan to make a blocking `ClusterHello` RPC that will return every container's rank + IP address in the cluster; we can figure out what to do with this information / how to expose it to the user later.